docs/templates/template_reference.md

```diff
@@ -815,9 +815,26 @@ The selected value can be changed in the profile (consult the actual variable fo
 
     -   **datatype** - data type of the sysctl value, eg. `int`.
 
-    -   **sysctlval** - value of the sysctl value, eg. `'1'`. If this
+    -   **sysctlval** - value of the sysctl value. This can be either an atomic
+        value, eg. `'1'`, or a list of values, eg. `['1','2']`. If this
         parameter is not specified, XCCDF Value is used instead.
 
+    -   **sysctlval_remediate** - the value that will be used in remediations.
+        If **sysctlval_remediate** is not specified, the template will use the
+        value of the **sysctlval** parameter in the remediations.
+        This parameter is mandatory when the **sysctlval** parameter is a list
+        because we need to know which of the values in the list the system
+        should be remedied to. When  the **sysctlval** parameter is not a list
+        this parameter is optional.
+
+    -   **sysctlval_wrong** - the value that is always wrong. This will be used
+        only in the test scenarios only if **sysctlval** is a list.
+
+    -   **missing_static_pass** - if set to `true` the check will pass if the
```

Comment:
Our project already has a notion of "missing_parameter_pass" - I would lean towards unifying the name. I don't find "missing_static_pass" significantly more descriptive.

Reply:
The intent behind this kind of parameter has always been somewhat controversial. Usually STIG profiles require them to be explicitly set. What we did for the SSHD parameters is that values are put under a different config file under the .d drop-in config directory. Unfortunately we cannot tailor this behavior per profile. For now it should be ok as we don't plan to use these rules across many different profiles, but we need to keep in mind this scenario.

Reply:
This is troublesome, sometimes customers want us to make checks [minimalistic](https://bugzilla.redhat.com/show_bug.cgi?id=2077531), sometimes they want them to be explicit (STIG). We could design content that would use an XCCDF value to parametrize this behavior, but that would be quite ugly (and out of the scope of this PR).